src/equations/numerical_fluxes.jl

```diff
@@ -333,13 +333,25 @@ struct FluxUpwind{Splitting}
   splitting::Splitting
 end
 
-@inline function (numflux::FluxUpwind)(u_ll, u_rr, orientation::Int, equations)
+@inline function (numflux::FluxUpwind)(u_ll, u_rr, orientation, equations)
   @unpack splitting = numflux
   fm = splitting(u_rr, Val{:minus}(), orientation, equations)
   fp = splitting(u_ll, Val{:plus}(),  orientation, equations)
   return fm + fp
 end
 
+# TODO: FD. Works properly for `splitting_lax_friedrichs` but needs further
+#       testing for other splittings.
+@inline function (numflux::FluxUpwind)(u_ll, u_rr, normal_direction::AbstractVector,
+                                       equations::AbstractEquations{2})
+  @unpack splitting = numflux
+  # Compute splitting in generic normal direction with specialized
+  # eigenvalues estimates calculated inside the `splitting` function
+  f_tilde_m = splitting(u_rr, Val{:minus}(), normal_direction, equations)
+  f_tilde_p = splitting(u_ll, Val{:plus}() , normal_direction, equations)
+  return f_tilde_m + f_tilde_p
+end
```

Comment:
Continuing the comment above, this is not consistent with the approach to compute numerical fluxes by rotation:
```julia
julia> using Revise; using Trixi

julia> equations = CompressibleEulerEquations2D(1.4)
┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
│ CompressibleEulerEquations2D                                                                     │
│ ════════════════════════════                                                                     │
│ #variables: ………………………………………………… 4                                                                │
│ │ variable 1: …………………………………………… rho                                                              │
│ │ variable 2: …………………………………………… rho_v1                                                           │
│ │ variable 3: …………………………………………… rho_v2                                                           │
│ │ variable 4: …………………………………………… rho_e                                                            │
└──────────────────────────────────────────────────────────────────────────────────────────────────┘

julia> u_ll = SVector(1.0, 0.2, -0.3, 2.0)
4-element SVector{4, Float64} with indices SOneTo(4):
  1.0
  0.2
 -0.3
  2.0

julia> u_rr = SVector(1.1, 0.3, -0.2, 2.1)
4-element SVector{4, Float64} with indices SOneTo(4):
  1.1
  0.3
 -0.2
  2.1

julia> normal_direction = SVector(0.3, -0.5)
2-element SVector{2, Float64} with indices SOneTo(2):
  0.3
 -0.5

julia> numflux = FluxUpwind(splitting_lax_friedrichs)
FluxUpwind(splitting_lax_friedrichs)

julia> numflux(u_ll, u_rr, normal_direction, equations) - FluxRotated(numflux)(u_ll, u_rr, normal_direction, equations)
4-element SVector{4, Float64} with indices SOneTo(4):
  0.00999999999999987
  0.002735925833319497
 -0.001804120520855823
  0.019080092708350427
```
This appears to be strange, doesn't it?